config/jobs/kubernetes/cloud-provider-vsphere/cloud-provider-vsphere-config.yaml

```diff
@@ -246,6 +246,39 @@ presubmits:
       testgrid-tab-name: pr-verify-test-coverage
       description: Shows the test coverage of the Golang sources
 
+  # Executes the e2e tests.
+  - name: pull-cloud-provider-vsphere-e2e-test
+    decorate: true
+    labels:
+      preset-dind-enabled: "true"
+      preset-cluster-api-provider-vsphere-e2e-config: "true"
```

Comment:
Could you add a comment here, noting that we are borrowing credentials from CAPV job

Reply:
sure